packages/terra-navigation-side-menu/src/MenuItem.module.scss

```diff
@@ -35,14 +35,17 @@
     }
 
     &.is-selected {
+      background-attachment: var(--terra-navigation-side-menu-item-selected-background-attachment);
       background-color: var(--terra-navigation-side-menu-item-selected-background-color, #f4f4f4);
+      background-image: var(--terra-navigation-side-menu-item-selected-background-image);
+      background-size: var(--terra-navigation-side-menu-item-selected-background-size);
       border-left: var(--terra-navigation-side-menu-item-selected-border-left, 0.5rem solid #007cc3);
       color: var(--terra-navigation-side-menu-item-selected-color);
       font-weight: var(--terra-navigation-side-menu-item-selected-font-weight, 600);
       padding-left: var(--terra-navigation-side-menu-item-selected-padding-left, 0.7143rem);
 
       &::before {
-        background-image: var(--terra-navigation-side-menu-item-selected-background-image);
+        background-image: var(--terra-navigation-side-menu-item-selected-before-background-image);
```

Comment:
This variable is pulled from [the OCS theme branch](https://github.com/cerner/terra-framework/blob/3152c95dd5c9c39f028a92d2d2e884549d282331/packages/terra-navigation-side-menu/src/MenuItem.module.scss#L90).